BIPs/2024-W21/BIP-103-Dec-April

```diff
@@ -0,0 +1,59 @@
+{
+    "version": "1.0",
+    "chainId": "1",
+    "createdAt": 1715865196870,
+    "meta": {
+      "name": "Transactions Batch",
+      "description": "",
+      "txBuilderVersion": "1.13.2",
+      "createdFromSafeAddress": "0x10A19e7eE7d7F8a52822f6817de8ea18204F2e4f",
+      "createdFromOwnerAddress": ""
+    },
+    "transactions": [
+      {
+        "to": "0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48",
+        "value": "0",
+        "data": null,
+        "contractMethod": {
+          "inputs": [
+            {
+              "name": "to",
+              "type": "address",
+              "internalType": "address"
+            },
+            {
+              "name": "value",
+              "type": "uint256",
+              "internalType": "uint256"
+            }
+          ],
+          "name": "transfer",
+          "payable": false
+        },
+        "contractInputsValues": {
+          "to": "0x58e6c7ab55Aa9012eAccA16d1ED4c15795669E1C",
+          "value": "53242000000"
+        },
+        "meta": {
```

Comment:
@MrOctopusK You have a second meta section here, that is conflicting with the frist one up top.  That is why it did not resolve.  Thanks for taking the time to outline everything in the payload meta.  

The fee breakdown may go better in the PR text than the meta.  I'll go ahead and restructre for you but have a look for next time :)

Reply:
I made this commit:
https://github.com/BalancerMaxis/multisig-ops/pull/976/commits/a9ae9a715dd000bd88e64e01532c9cb8e9909299

If you have a more readable/tabular/etc breakdown of the fee data it could be nice to put in the description up top.  Just click the 2 dots on the top left and hit edit.

Reply:
Also we're not well setup to validate things from a fork.  I think you can make a PR directly in this repo can't you?  Let me know if you want to hop on a quick screenshare next time.

Reply:
See #977 for further processing.  Would be nice @MrOctopusK if you could just confirm everything there looks ok in a comment, and maybe provide a screenshot of a breakdown of the fees in a more human readable form (also in the comment).